conda_mirror/diff_tar.py

```diff
@@ -0,0 +1,191 @@
+import os
```

Comment:
Can you please add a module-level docstirng

Reply:
Just did.